rest_framework/pagination.py

```diff
@@ -377,14 +391,33 @@ class LimitOffsetPagination(BasePagination):
     http://api.example.org/accounts/?limit=100
     http://api.example.org/accounts/?offset=400&limit=100
     """
-    default_limit = api_settings.PAGE_SIZE
     limit_query_param = 'limit'
     limit_query_description = _('Number of results to return per page.')
     offset_query_param = 'offset'
     offset_query_description = _('The initial index from which to return the results.')
-    max_limit = None
```

Comment:
Can we avoid changing the style here?

```python
max_limit = api_settings.MAX_PAGE_SIZE
```

Perfer minimal impact footprints for PRs.

Reply:
It's related to https://github.com/encode/django-rest-framework/pull/8993. Without it we can't properly test it, since this setting will be set during init

Reply:
Hrm... not sure? If you'd like to take that approach then I'd like to handle #8993 separately, as a precursor to this.
Unclear to me why that'd be different to attributes [we use in other cases](https://github.com/encode/django-rest-framework/blob/master/rest_framework/views.py#L108-L115).